frontend/src/views/SettingWorkspaceGeneral.vue

```diff
@@ -4,25 +4,34 @@
     <BrandingSetting />
     <SecuritySetting />
     <AIAugmentationSetting />
+    <AnnouncementSetting v-if="isEnterprisePlan" />
   </div>
 </template>
 
 <script lang="ts" setup>
 import { storeToRefs } from "pinia";
-import { onMounted } from "vue";
+import { onMounted, computed } from "vue";
 import {
   BrandingSetting,
   SecuritySetting,
   NetworkSetting,
   AIAugmentationSetting,
+  AnnouncementSetting,
 } from "@/components/GeneralSetting";
-import { useActuatorV1Store } from "@/store";
+import { useActuatorV1Store, useSubscriptionV1Store } from "@/store";
 import { useSettingV1Store } from "@/store/modules/v1/setting";
+import { PlanType } from "@/types/proto/v1/subscription_service";
 
 const actuatorStore = useActuatorV1Store();
+const subscriptionStore = useSubscriptionV1Store();
 
 onMounted(async () => {
   await useSettingV1Store().fetchSettingList();
 });
 const { isSaaSMode } = storeToRefs(actuatorStore);
+
+const isEnterprisePlan = computed((): boolean => {
```

Comment:
We should always show the feature. Otherwise, what if the user has enabled announcement beforehand? You can check the  "security" setting under the "general" section for more patterns.